config.toml

```
@@ -111,7 +84,7 @@ timeout = 300000
     parent = "docs"
     weight = 8
     identifier = "eip"
-    url = "/components/latest/eips/enterprise-integration-patterns.html"
+    url = "/manual/latest/enterprise-integration-patterns.html"
```

I think, this should be:
```suggestion
    url = "/documentation.html"
```